channels/sync_runner.go

```diff
@@ -123,6 +123,9 @@ func NewSyncRunner(funcSource string) (*SyncRunner, error) {
 		return nil, err
 	}
 
+	runner.JSRunner.SetLogCallback(func(s string) { base.Infof(base.KeySyncFuncLog, base.UD(s).Redact()) })
```

Comment:
This is stylistic but because the `JSRunner` struct is embedded, we can call `SetLogCallback` directly on runner.